postgres-appliance/Dockerfile.build

```diff
@@ -375,7 +375,8 @@ RUN export DEBIAN_FRONTEND=noninteractive \
                 python3-psycopg2 python3-pyasn1-modules python3-rsa python3-dateutil ${BUILD_PACKAGES} \
 
     && pip3 install setuptools --upgrade \
-
+    && pip3 install filechunkio \
```

Comment:
please move it to the line 385: `pip3 install filechunkio envdir wal-e[aws,google]==$WALE_VERSION`